OracleWebCenterSites/README.md

```diff
@@ -295,7 +295,7 @@ b. Run the following command and pass the environment file name as a parameter:
 ```
 Sample command:
 ```
-   $ docker run -d -t --name WCSitesManagedContainer --network=WCSitesNet --volumes-from WCSitesAdminContainer -p 7002:7002 -p 9002:9002 --env-file ./wcsitesserver.env.list oracle/wcsites:12.2.1.3 /bin/bash -c "/u01/oracle/sites-container-scripts/startSitesServer.sh; /bin/bash"
+   $ docker run -d -t --name WCSitesManagedContainer --network=WCSitesNet --volumes-from WCSitesAdminContainer -p 7002:7002 -p 9002:9002 --env-file ./wcsitesserver.env.list oracle/wcsites:12.2.1.4 /bin/bash -c "/u01/oracle/sites-container-scripts/startSitesServer.sh; /bin/bash"
```

Comment:
Why are you providing commands here? Why isn't your Dockerfile running the `startSiteServer.sh` script itself via the `CMD` directive (which runs as `bash -c` anyway)? Also, why are you starting `/bin/bash` in a detached container? If you need the container to run, just make sure the `startSiteServer.sh` script doesn't exit. You need to output the logs to `stdout` anyway for `docker logs` to work.